CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Changed
+- Technical - Upgrade `mongoose` to the latest version in template.
```

Comment:
This is not a "Technical" change from my point of view.
It is about "Vulnerabilities"/"Security".

"Upgrade `mongoose` to the latest version in template" is something you can already read in the changelog, so I think we should explain the intent.

```suggestion
- Command Generate - Ensure that generated project on MongoDB use the latest `mongoose` dependency version for security reasons.
```